File: smoketest/sumas.js
const readline = require("node:readline");

const rl = readline.createInterface({
  input: process.stdin,
});

rl.on("line", (input) => {
  console.log(
    input
      .trim()
      .split(" ")
      .map((x) => parseInt(x))
      .reduce((acc, x) => acc + x, 0)
  );
});

File: smoketest/sumas.rs
use std::io;

fn main() -> io::Result<()> {
    let mut line = String::new();

    io::stdin().read_line(&mut line)?;
    let mut sum: i64 = 0;
    for s in line.trim().split(" ") {
        sum += i64::from_str_radix(s, 10).unwrap();
    }
    println!("{}", sum);

    Ok(())
}
